### DISCUSSION TITLE
LANConnect

### DISCUSSION THREAD

**Discussion by zsr-lukezhang** [win12-online/win12#495] [General]
有任何人会 WinUI 吗
我知道这似乎与 Win12 毫无关系（

但是。。。我需要帮助🙃
[LANConnect](https://github.com/zsr-lukezhang/LANConnect)

**Comment by lingbopro** [comment]
WinUI我可能会一点（？

我会一点点WPF

**Reply by zsr-lukezhang** [reply]
好的，去改代码吧（？

**Reply by zsr-lukezhang** [reply]
@lingbopro 不 Fork 的吗

**Reply by lingbopro** [reply]
这不没时间嘛

**Comment by zsr-lukezhang** [comment]
if (WinUI == WPF)
{
      Debug.WriteLine ("Error when processing")
}
else
{
      Debug.WirteLine ("Fake information dectected")
}

[doge]

**Reply by zsr-lukezhang** [reply]
还少了分号（）

**Reply by lingbopro** [reply]
确实`WinUI != WPF`

**Comment by starry-source** [comment]
会一点 winui2, winui3 就算了，设计器都没有

**Reply by zsr-lukezhang** [reply]
不是有热重载吗

**Reply by lingbopro** [reply]
没设计器这个确实，我之前做过，特别不方便

**Reply by zsr-lukezhang** [reply]
![image](https://github.com/user-attachments/assets/37fc9439-ff03-4053-baad-6658c72de85a)

**Reply by ProgramCX** [reply]
> 会一点 winui2, winui3 就算了，设计器都没有

有的吧，但是我的出现了一些抽象的错误
![image](https://github.com/user-attachments/assets/7129edcf-fc9e-47e3-930a-906b195f72ab)

**Reply by zsr-lukezhang** [reply]
WinUI 3 没有的
@ProgramCX 只有 WPF Winform VBA 之类的有设计器

**Reply by zsr-lukezhang** [reply]
> WinUI 3 没有的 @ProgramCX 只有 WPF Winform VBA 之类的有设计器

重启一下VS设计器会自动消失的

**Reply by ProgramCX** [reply]
啊这

**Comment by lingbopro** [comment]
对了，**我个人觉得** 这个项目可以尝试一下 [Avalonia](https://github.com/AvaloniaUI/Avalonia)，也是用 C# 写桌面应用，而且跨平台，配合 [FluentAvalonia](https://github.com/amwx/FluentAvalonia) 这个库也可以实现像 WinUI3 这样的主题 :P

**Reply by zsr-lukezhang** [reply]
> 对了，**我个人觉得** 这个项目可以尝试一下 [Avalonia](https://github.com/AvaloniaUI/Avalonia)，也是用 C# 写桌面应用，而且跨平台，配合 [FluentAvalonia](https://github.com/amwx/FluentAvalonia) 这个库也可以实现像 WinUI3 这样的主题 :P

我就是喜欢 这玩意 主要是因为 ~~Copilot 写这玩意很好~~ **[WinUI 3 Gallery](https://github.com/microsoft/WinUI-Gallery)** 可以直接抄
第三方的应该没有微软官方的好用吧

**Reply by zsr-lukezhang** [reply]
@lingbopro ~~实际上就是因为不想重写~~

**Reply by Mbilse** [reply]
用chatgpt吧...
我有亿个帐号

**Comment by tjy-gitnub** [comment]
👇这个👇👇
  WPF UI
  
  超好用

**Reply by lingbopro** [reply]
我的评价是不如WinUI

**Reply by tjy-gitnub** [reply]
不然，wpfui有设计器，winui没有。
